lib/Metrc.rb

```diff
@@ -1,16 +1,18 @@
+# rubocop:disable Naming/FileName,Lint/MissingCopEnableDirective
+
 require 'httparty'
 require 'Metrc/constants'
 require 'Metrc/errors'
 require 'Metrc/configuration'
 require 'Metrc/client'
-require "Metrc/version"
+require 'Metrc/version'
 
 module Metrc
   def self.configuration
     @configuration ||= Configuration.new
   end
 
-  def self.configure(&block)
+  def self.configure(&block) # rubocop:disable Lint/UnusedMethodArgument
```

Comment:
another way to do this is to add a `_` in front of the variable name... `_block` or `self.configure(&_block)`

Reply:
Fixed! Thanks for this.